elastic/kibana
# [TSVB] Add support for histogram type

## Summary

This PR adds support for histogram types to the following TSVB aggregations:
- average
- sum
- value_count
- percentiles
- percentile ranks
- filter ratio (metric aggregations)

### Testing Data

1. Run Metricbeat locally
2. Use the [this script](https://gist.github.com/simianhacker/95811e4c717fee10b01e39055448a8c3#file-create_histogram_data-js
) to create some histogram fields. 

See #52426 for details
Fixes #70984

### Checklist
